From 4846d3cf355075fd59f2d7c8019f188df7dd75cc Mon Sep 17 00:00:00 2001
From: Rob Bradford <rob@linux.intel.com>
Date: Mon, 28 Jan 2013 12:29:34 +0000
Subject: wayland: Fix configure flag around cairo egl support

The intention was for this to default to off. However just using this option
(whether to enable or disable it) would turn it on.

Bugzilla: https://bugzilla.gnome.org/show_bug.cgi?id=688534

Thanks to Quentin Glidic for spotting this issue.
---
diff --git a/configure.ac b/configure.ac
index b8a5c68..fe0cb89 100644
--- a/configure.ac
+++ b/configure.ac
@@ -324,7 +324,8 @@ AC_ARG_ENABLE(quartz-relocation,
 AC_ARG_ENABLE(wayland-cairo-gl,
               AS_HELP_STRING([--enable-wayland-cairo-gl],
                              [enable the use of Cairo GL in the Wayland backend]),
-              [enable_wayland_cairo_gl=yes])
+              [],
+              [enable_wayland_cairo_gl=no])
 
 
 cairo_backends=
--
cgit v0.9.1
 
